package.json

```diff
@@ -32,11 +32,11 @@
     "eslint-plugin-jest": "^25.3.4",
     "eslint-plugin-prettier": "^3.4.0",
     "graphql": "^15.5.1",
-    "inquirer": "^8.2.1",
+    "inquirer": "^8.2.4",
     "jest": "^27.4.5",
     "jest-docblock": "^27.4.0",
     "markdown-js": "^0.0.4",
-    "marked": "^4.0.10",
+    "marked": "^2.0.1",
```

Comment:
Using this version as opposed to `4.0.10`, what are you observing? Seeing that it is taking a downgrade, wondering what might be different.

Reply:
I am sorry sir , actually I tried to using `yarn install ` command but It was not working to install packages . then I installed using the `yarn setup` (which runs an installation script) command which installed the previous version. 
i'll change it right away